ArduPlane/quadplane.cpp

```diff
@@ -1977,6 +1977,24 @@ void QuadPlane::motors_output(bool run_rate_controller)
         attitude_control->rate_controller_run();
     }
 
+    /* Delay for ARMING_DELAY_MS after arming before allowing props to spin:
+       1) for safety (OPTION_DELAY_ARMING)
+       2) to allow motors to return to vertical (OPTION_DISARMED_TILT)
+     */
+    if ((options & OPTION_DISARMED_TILT) || (options & OPTION_DELAY_ARMING)) {
+        constexpr uint32_t ARMING_DELAY_MS = 2000;
+        if (hal.util->get_delay_arming()) {
+            if (AP_HAL::millis() - hal.util->get_last_armed_change() < ARMING_DELAY_MS) {
+                // delay motor start after arming
+                motors->set_desired_spool_state(AP_Motors::DesiredSpoolState::SHUT_DOWN);
+                motors->output();
+                return;
+            } else {
+                hal.util->reset_delay_arming();
```

Comment:
I don't think this belongs at the hal level, it creates a dependency on ARMING_DELAY_MS in the hal

Reply:
OK, won't consider consolidating the arming delay with copter